### PR TITLE
Gimbap rich media

### PR DESCRIPTION
**Pull request by Calum-Campbell**
- Gimbap richmedia native template
- The original css inherited from the fs-healine mixin which is now gone. As such I have added some scss to the src/manual-gimbap-richmedia/index.scss file.

image

@regiskuckaertz @guardian/commercial-dev

### CODE REVIEW

File: src/_shared/scss/_gimbap-richmedia.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.gimbap-richmedia__title {
+  @extend %gimbap-text-styling;
+
+  font-size: 1.25rem;
+  font-family: "Guardian Egyptian Web","Guardian Text Egyptian Web",Georgia,serif;
+
+  font-weight: bold;
+  line-height: .8;
```
Comment (regiskuckaertz) [inline src/_shared/scss/_gimbap-richmedia.scss]: The 4 lines above can be rewritten:

```
font: bold 1.25rem/.8 $f-serif-text;
```

File: src/manual-gimbap-richmedia/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                <p class="gimbap-richmedia__headline">[%Offer1Headline%]</p>
+                <span class="gimbap-richmedia__meta gimbap-richmedia__meta--date metaHide"><span class="gimbap-richmedia__icon inline-icon-clock inline-commercial">{{#svg}}icon-clock{{/svg}} [%Offer1Date%]</span></span>
+                <span class="gimbap-richmedia__meta gimbap-richmedia__meta--place metaHide"><span class="gimbap-richmedia__icon inline-icon-location inline-commercial">{{#svg}}icon-location{{/svg}}</span></span>
+                <span class="gimbap-richmedia__price gimbap-richmedia__meta--price metaHide">&#163; [%Offer1Price%]</span>
+                <span class="gimbap-richmedia__meta gimbap-richmedia__meta--saving metaHide"><span class="gimbap-richmedia__icon inline-icon-basket inline-commercial">{{#svg}}icon-basket{{/svg}} [%Offer1Discount%]</span></span>
+                <img class="gimbap__image gimbap-richmedia__img" src="[%Offer1Image%]" />
+            </a>
+            <a class="gimbap gimbap--simple gimbap--richmedia gimbap--tone-[%ComponentTone%] [%GimbapEffects%]" href="%%CLICK_URL_UNESC%% [%Offer2Url%]" data-link-name="[%Offer2Title%] | Offer 2">
```
Comment (regiskuckaertz) [inline src/manual-gimbap-richmedia/index.html]: No blank space in the `href` attribute

File: src/manual-gimbap-richmedia/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<aside class="gimbap-wrap gimbap-wrap--tone-[%ComponentTone%]" role="complementary" data-link-name="creative | gimbap richmedia | [%TrackingId%]">
+    <div class="gimbap-wrap__box">
+        <header class="gimbap-wrap__header">
+            <h3 class="gimbap-wrap__title">
+                <a class="gimbap-wrap__logo gimbap-wrap__logo--[%ComponentTone%] gimbap-logo gimbap-logo--tone-[%ComponentTone%]" href="%%CLICK_URL_UNESC%%" data-link-name="title"></a>
```
Comment (regiskuckaertz) [inline src/manual-gimbap-richmedia/index.html]: No link after the click macro?

File: src/manual-gimbap-richmedia/index.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    function removeHideClass(div){
+        div.classList.remove('metaHide');
+    }
+
+    function insertHeaderSvg(div) {
+        div.insertAdjacentHTML('afterbegin', logoSvgs[componentTone]);
+    }
+
+    function addEffectClass(div) {
+        div.className += " gimbap--effects";
```
Comment (regiskuckaertz) [inline src/manual-gimbap-richmedia/index.js]: `div.classList.add("gimbap--effects")`

Comment (regiskuckaertz) [inline src/manual-gimbap-richmedia/index.js]: Although once again, you could instead have a `gimbap-effects--yes` class in your CSS and there would be no need for javascript.

File: src/manual-gimbap-richmedia/index.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            if(index < array.length/2){
+                removeHideClass(Array.from(document.getElementsByClassName('gimbap-richmedia__meta--' + pair[0]))[0])
+            }else{
+                removeHideClass(Array.from(document.getElementsByClassName('gimbap-richmedia__meta--' + pair[0]))[1])
+            }
+        }
+    }
+
+    function removeHideClass(div){
+        div.classList.remove('metaHide');
```
Comment (regiskuckaertz) [inline src/manual-gimbap-richmedia/index.js]: wrap DOM mutations in `write` calls

File: src/manual-gimbap-richmedia/index.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
+@import '_core';
+@import '_gimbap-branding';
+@import '_gimbap-logo';
+@import '_gimbap-wrap';
+@import '_gimbap';
+@import '_gimbap-richmedia';
+
+$gimbap-larger-text: "Guardian Egyptian Web","Guardian Text Egyptian Web",Georgia,serif;
```
Comment (regiskuckaertz) [inline src/manual-gimbap-richmedia/index.scss]: Check `$f-serif-text` and `$f-serif-headline` in `_core`

**Comment by Calum-Campbell** [comment]
@regiskuckaertz updates done, have also update the gimbap-simple to use css for the gimbap--eftects.

**Pull request merged**